packages/forms/frontend/sirius-components-widget-reference/src/SingleValuedReferencePropertySection.tsx

```diff
@@ -0,0 +1,195 @@
+    if (!loading) {
+      if (error) {
+        addErrorMessage('An unexpected error has occurred, please refresh the page');
+      }
+      if (data) {
+        const { editSingleValuedReference } = data;
+        if (isErrorPayload(editSingleValuedReference) || isSuccessPayload(editSingleValuedReference)) {
+          addMessages(editSingleValuedReference.messages);
+        }
+      }
+    }
+  }, [loading, error, data]);
+
+  const handleDragEnter: React.DragEventHandler<HTMLDivElement> = (event) => {
+    event.preventDefault();
+  };
+
+  const handleDragOver: React.DragEventHandler<HTMLDivElement> = (event) => {
+    event.preventDefault();
+  };
+
+  const handleDrop = (event: React.DragEvent) => {
+    event.preventDefault();
+    if (readOnly) {
+      addErrorMessage('This widget is currently read-only');
+    } else {
+      const dragSourcesStringified = event.dataTransfer.getData(DRAG_SOURCES_TYPE);
+      if (dragSourcesStringified) {
+        const sources = JSON.parse(dragSourcesStringified);
+        if (Array.isArray(sources) && sources.length === 1) {
+          const entry = sources[0] as SelectionEntry;
+          if (entry.kind.startsWith('siriusComponents://semantic?')) {
+            const newValueId = entry.id;
+            const variables = {
+              input: {
+                id: crypto.randomUUID(),
+                editingContextId,
+                representationId: formId,
+                referenceWidgetId: widget.id,
+                newValueId,
+              },
+            };
+            editSingleValuedReference({ variables });
+          }
+        }
+      }
+    }
+  };
+
+  return (
+    <div>
+      <PropertySectionLabel
+        editingContextId={editingContextId}
+        formId={formId}
+        widget={widget}
+        subscribers={subscribers}
+        data-testid={widget.label}
+      />
+      <div onDragEnter={handleDragEnter} onDragOver={handleDragOver} onDrop={handleDrop}>
+        <List dense>
+          {widget.referenceValue ? (
+            <ListItem>
+              <ListItemIcon>
+                {widget.referenceValue.iconURL ? (
+                  <img width="16" height="16" alt={''} src={httpOrigin + widget.referenceValue.iconURL} />
+                ) : null}
+              </ListItemIcon>
+              <ListItemText data-testid={`reference-value-${widget.referenceValue.id}`}>
+                {widget.referenceValue.label}
+              </ListItemText>
+              <IconButton
+                aria-label="clearReference"
+                disabled={readOnly}
+                onClick={() => onDelete()}
+                data-testid={`clear-reference-${widget.id}`}>
+                <DeleteIcon />
+              </IconButton>
+            </ListItem>
+          ) : (
+            <ListItem>
+              <ListItemText data-testid="reference-value-none">None</ListItemText>
+            </ListItem>
+          )}
+        </List>
+      </div>
+      <Snackbar
```

Comment:
The other widgets  (select, textfield...) don't use snackbar anymore, why this one (and  other reference property sections) still needs it?

Reply:
This file is actually dead code that I forgot to delete.
The single widget/property section if in `ReferencePropertySection`, which indeed has the same issue, and I will fix it there.